partials.md

```diff
@@ -0,0 +1,50 @@
+# Partials
+
+Maud does not have a built-in concept of partials or sub templates. Instead, you can compose your markup with any function that returns `Markup`.
+
+The following uses a `header` and `footer` function that are used in the `page` function to return a final result.
+
+Here is a page that has a dynamic page title, a body and a footer with an RSS link:
+
+```rust
+extern crate maud;
+
+use self::maud::{DOCTYPE, html, Markup};
+
+/// A basic header with a dynamic `page_title`.
+fn header(page_title: &str) -> Markup {
+    html! {
+        (DOCTYPE)
+        html {
+            meta charset="utf-8";
+            title { (page_title) }
+        }
+    }
+}
+
+/// A static footer.
+fn footer() -> Markup {
+    html! {
+        footer {
+            span {
+                a href="rss.atom" { "RSS Feed" }
+            }
+        }
+    }
+}
+
+/// The final Markup, including `header` and `footer`.
+pub fn page(title: &str) -> Markup {
```

Comment:
I think it's worth demonstrating that functions can accept `Markup` values as well as return them.

See the first few lines of https://github.com/lfairy/karkinos/blob/master/src/views.rs

What do you think?

Reply:
That makes sense :+1: